package.json

```diff
@@ -65,6 +64,7 @@
     "css-loader": "0.28.x",
     "eslint": "^7.31.0",
     "eslint-config-prettier": "^8.3.0",
+    "eslint-import-resolver-typescript": "^2.7.1",
```

Comment:
is this import for ` 'plugin:import/typescript',`

Reply:
yes, it is for that only [ref](https://github.com/import-js/eslint-plugin-import#typescript)